src/test/java/com/iexec/worker/compute/ComputeManagerServiceTests.java

```diff
@@ -357,12 +360,14 @@ void shouldRunStandardPostCompute() {
     @EnumSource(value = ReplicateStatusCause.class, names = "POST_COMPUTE_.*", mode = EnumSource.Mode.MATCH_ALL)
     void shouldRunStandardPostComputeWithFailureResponse(ReplicateStatusCause statusCause) {
         final TaskDescription taskDescription = createTaskDescriptionBuilder(false).build();
-        PostComputeResponse postComputeResponse = PostComputeResponse.builder().exitCauses(List.of(statusCause)).build();
+        PostComputeResponse postComputeResponse = PostComputeResponse.builder()
+                                                    .exitCauses(List.of(new WorkflowError(statusCause))).build();
```

Comment:
The indentation of the new lines seems really big
```suggestion
        final PostComputeResponse postComputeResponse = PostComputeResponse.builder()
                .exitCauses(List.of(new WorkflowError(statusCause)))
                .build();
```

Reply:
Resolved in [be8f3a2](https://github.com/iExecBlockchainComputing/iexec-worker/pull/662/commits/be8f3a2849b8b721802766a6bf9cab3aeab4feda)